forms-flow-web/src/components/Modeller/ExampleModels/pizza-collaboration.bpmn

```diff
@@ -0,0 +1,383 @@
+<?xml version="1.0" standalone="yes"?>
```

Comment:
Are we using these inside ? Or can use these to write the test cases

Reply:
They were only used for my own testing. I have removed them for now